app/src/main/java/seveida/firetvforreddit/subreddit/domain/objects/UserDetails.java

```diff
@@ -0,0 +1,27 @@
+package seveida.firetvforreddit.subreddit.domain.objects;
+
+import java.net.URI;
+import java.text.DateFormat;
+import java.util.List;
+
+import androidx.annotation.NonNull;
+
+public class UserDetails {
+
+
+    @NonNull public final UserMetadata userMetadata;
+    @NonNull public final URI avatar;
```

Comment:
This is the wrong `URI` class.  It's confusing, but we want `Uri` (with an Android package), not `URI` (with a java package).

Reply:
good catch!